source/_components/waterfurnace.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "Waterfurnace"
+description: "Instructions on how to integrate Waterfurnace Geothermal System into Home Assistant."
+date: 2018-01-19 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: waterfurnace.png
+ha_category: Hub
+ha_release: 0.62
+ha_iot_class: "Cloud Polling"
+---
+
+The `Waterfurnace` component communicates with the Waterfurnace
+Symphony website's websocket to show you many of the sensors in your
+system. While not an official API, this is the same backend the
+Symphony website is based on, and should be reasonably stable.
+
+The sensors provided include:
+ - Thermostat Setpoint
+ - Thermostat Current Temp
+ - Leaving Air Temp
+ - Entering Water Loop Temp
+ - Current Humidity
+ - Current Humidity Setpoint
+ - Total system power (in Watts)
+ - Furnace Mode
+
+To use Waterfurnace in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+waterfurnace:
+  username: email
```

Comment:
Maybe use some example data instead of placeholders?

```yaml
waterfurnance:
  username: john@example.com
  password: supersecret
  unit: UJ1234
```